composer.json

```diff
@@ -8,17 +8,23 @@
     "issues": "https://github.com/leth/PHP-IPAddress/issues"
   },
   "require": {
+    "php": ">=8.1",
```

Comment:
This definitely requires a new major release, since it's a big break in backwards compatibility.

Reply:
Okay, don't sweat this one, I'm going to redirect this into a new 2.0 branch, so I'll do a separate PR for that change.